Rawl_B502/forshluinti_úa_náir.ttl

```diff
@@ -0,0 +1,462 @@
+    rel:descendantOf <#FergusLáebderc>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Lochan>.
+
+<#Énna>
+    a foaf:Person;
+    irishRel:nomName "Énna";
+    rdfs:comment "ó Chill Manach Escrach";
+    rel:descendantOf <#FergusLáebderc>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Enna-dbafa850>.
+
+<#MoSenóc>
+    a foaf:Person;
+    irishRel:nomName "Mo Senóc";
+    rdfs:comment "Mugnai";
+    rel:descendantOf <#FergusLáebderc>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#MuShenoc>.
+
+<#MoChóeme>
+    a foaf:Person;
+    irishRel:nomName "Mo Chóeme";
+    rdfs:comment "Tíre Dá Glas";
+    rel:descendantOf <#FergusLáebderc>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#MoChoeme>.
+
+<#Cóemán>
+    a foaf:Person;
+    irishRel:nomName "Cóemán";
+    rdfs:comment "Enaig Thruimm";
+    rel:descendantOf <#FergusLáebderc>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Coeman>.
+
+<#Petrán>
+    a foaf:Person;
+    irishRel:nomName "Petrán";
+    rdfs:comment "ó Chill Lainni";
+    rel:descendantOf <#FergusLáebderc>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Petran>.
+
+<#Éitcheáin>
+    a foaf:Person;
+    irishRel:genName "Éitcheáin";
+    irishRel:nomName "Éitcheán";
+    rel:childOf <#Maine>;
+    rel:employerOf [
+      a foaf:Person;
+      foaf:title "epscop"@sga, "bishop"@eng;
+      rel:descendantOf <#FergusLáebderc>
+    ];
+    rdfs:comment "écis ó Chluain Fota Báetáin Aba";
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Étchaín>.
+
+<#Maine>
+    a foaf:Person;
+    irishRel:genName "Maine";
+    irishRel:nomName "Maine";
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Mane-69a21e90>
+
+<#NathÍ>
+    a foaf:Person;
+    irishRel:nomName "Nath Í";
+    foaf:title "epscop"@sga, "bishop"@eng;
+    rdfs:comment "h-i Cúil Fothirbi";
+    rel:childOf <#Senaich>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#NathÍ-f9faa250>.
+
+<#Senaich>
+    a foaf:Person;
+    irishRel:genName "Senaich";
+    irishRel:nomName "Senach";
+    rel:childOf <#Fergusa>.
+
+<#Fergusa>
+    a foaf:Person;
+    irishRel:genName "Fergusa";
+    irishRel:nomName "Fergus";
+    rel:descendantOf <#FergusLáebderc>.
+
+<#MoAcrau>
+    a foaf:Person;
+    irishRel:nomName "Mo Acrau";
+    rel:childOf <#Senáin>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#MoAcru>.
+
+<#Senáin>
+    a foaf:Person;
+    irishRel:genName "Senáin";
+    irishRel:nomName "Senán";
+    rel:descendantOf <#FergusLáebderc>.
+
+<#Connláed>
+    a foaf:Person;
+    irishRel:nomName "Connláed";
+    foaf:nick "cráibdech"@sga;
+    foaf:title "epscop"@sga, "bishop"@eng;
+    rdfs:comment "Cille Dara";
+    rel:descendantOf <#FergusLáebderc>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Condlaed>.
+
+<#Chaili>
+    a foaf:Person;
+    irishRel:genName "Chaili";
+    irishRel:nomName "Caile";
+    rdfs:comment "[Sluinti olchena] Úi Chaili";
```

Comment:
I didn't add it, exactly. It's in the text. This is a list of "other kindreds" (of Uí Náir - the less than foremost kindreds, presumably). Again (see previous comment), it is a list of population groups that would lack clear context without such annotation.

Ultimately, we might have to devise relationships between population group URIs to handle this sort of material. But, for now, I thought rdfs:comment was sufficiently free-style to accomodate it?